virt/release_notes/virt-4-14-release-notes.adoc

```diff
@@ -122,15 +124,16 @@ The SVVP Certification applies to:
 
 * Cluster administrators can now enable automatic subscription for {op-system-base-full} virtual machines in the {VirtProductName} xref:../../virt/getting_started/virt-web-console-overview.adoc#overview-settings-cluster_virt-web-console-overview[web console].
```

Comment:
Github won't let me comment on the correct lines, but if the Installation and Storage sections are going to be empty, please comment out the section headings

Reply:
(We can always uncomment the commented-out sections if we need to add something later)